website/docs/api/doc.md

```diff
@@ -445,7 +445,7 @@ Mark a span for merging. The `attrs` will be applied to the resulting token (if
 they're context-dependent token attributes like `LEMMA` or `DEP`) or to the
 underlying lexeme (if they're context-independent lexical attributes like
 `LOWER` or `IS_STOP`). Writable custom extension attributes can be provided as a
-dictionary mapping attribute names to values as the `"_"` key.
+dictionary mapping attribute name to values as the `"_"` key.
```

Comment:
I think "names" was correct: the dictionary maps attribute names to values. This sentence parses quite badly though, so perhaps we need to rewrite it into something like 
"... can be provided using the `"_"` key and specifying a dictionary that maps attribute name to values"